app/controllers/courses_controller.rb

```diff
@@ -2,7 +2,7 @@ class CoursesController < ApplicationController
   before_filter :load_course, except: [:index]
 
   def index
-    @courses = Course.order(:name).paginate(page: params[:page], per_page: 10)
+    @courses = Course.order(:name).paginate(page: params[:page], per_page: 5).includes(:institute).all
```

Comment:
The trailing `.all` is not needed, and as a preference, I like to put the `includes` at the beginning of the query

```suggestion
    @courses = Course.includes(:institute).order(:name).paginate(page: params[:page], per_page: 5)
```